objects/ClueCounterSwapper.d919d6.json

```diff
@@ -63,15 +63,15 @@
   "Sticky": true,
   "Tooltip": false,
   "Transform": {
-    "posX": -50.5,
-    "posY": 1.581,
+    "posX": -50.9,
+    "posY": 1.51,
     "posZ": 0,
     "rotX": 0,
     "rotY": 270,
     "rotZ": 0,
-    "scaleX": 0.420461982,
+    "scaleX": 0.4,
     "scaleY": 0.008194907,
```

Comment:
should scale be rounded? if so to what precision

Reply:
I did so intentionally, looks better to me ingame by having the same x and z scale

Reply:
added additional rounding for scaleY